src/Sequence.js

```diff
@@ -2,6 +2,8 @@
  * Sequence class
  */
 
+var Note = Note || ( require && require('./Note') );
```

Comment:
`let {Note, Sequence} = require('tinymusic');` is way more normal than `require('./node_modules/tinymusic/dist/TinyMusic.js');` (and relying on global scope mutations).

And yeah, lemme rephrase that check for require to use typeof like the `module` check at the bottom.